src/Generator/CreateTargetStatementsGenerator.php

```diff
@@ -195,21 +170,17 @@ private function constructorArguments(GeneratorMetadata $metadata): array
      *  }
      * ```
      *
-     * @return array{Stmt, Arg, string}|null
+     * @return array{Stmt, Arg, string}|array{null, null, null}
      */
-    private function constructorArgument(GeneratorMetadata $metadata, PropertyMetadata $propertyMetadata): ?array
+    private function constructorArgument(GeneratorMetadata $metadata, PropertyMetadata $propertyMetadata, \ReflectionParameter $parameter): array
     {
-        if (null === $propertyMetadata->target->writeMutatorConstructor || null === ($parameter = $propertyMetadata->target->writeMutatorConstructor->parameter)) {
-            return null;
-        }
-
         $variableRegistry = $metadata->variableRegistry;
         $constructVar = $variableRegistry->getVariableWithUniqueName('constructArg');
         $fieldValueExpr = $propertyMetadata->source->accessor?->getExpression($variableRegistry->getSourceInput());
 
         if (null === $fieldValueExpr) {
             if (!($propertyMetadata->transformer instanceof AllowNullValueTransformerInterface)) {
-                return null;
+                return [null, null, null];
```

Comment:
A bit cursed, why this change ? `null` would be still fine there no ?

Reply:
Don't change really anything, and phpstan works better that way, so i made it like this